### PR TITLE
Update ome-faq.yml

### PR DESCRIPTION
**Pull request by samschan-ms**
MInor update to clarify pdf encryption

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9

File: microsoft-365/compliance/ome-faq.yml [hunk 1]
```diff
@@ -135,11 +135,9 @@ sections:
       - question: |
           Are PDF file attachments supported?
         answer: |
-          The short answer is yes! If enabled, PDF encryption allows you to protect sensitive PDF documents through secure communication or secure collaboration. When you send an email, the Office 365 service encrypts PDF file attachments. The Outlook client doesn't encrypt PDF file attachments.
+          The short answer is yes! If enabled in Exchange Online, PDF encryption allows you to protect sensitive PDF documents attached to email. When you send an email, the Office 365 service encrypts PDF file attachments for Outlook on the web, Outlook Mac, Outlook for iOS, and Outlook for Android. You can encrypt PDFs you send without any more steps.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/ome-faq.yml]:
```suggestion
          The short answer is yes! If enabled in Exchange Online, PDF encryption allows you to protect sensitive PDF documents attached to emails. When you send an email, the Office 365 service encrypts PDF file attachments for Outlook on the web, Outlook for Mac, Outlook for iOS, and Outlook for Android. You can encrypt PDFs you send without any more steps.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @samschan-ms, kindly commit the minor suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@KCCross Please merge this PR. Thanks

**Pull request merged**